network_analysis/routing-around-barriers/README.metadata.json

```diff
@@ -1,34 +1,42 @@
 {
-  "category": "Network analysis",
-  "description": "Find a route with multiple stops and barriers.",
-  "ignore": false,
-  "images": [
-    "RoutingAroundBarriers.png"
-  ],
-  "keywords": [
-    "barriers",
-    "best sequence",
-    "directions",
-    "maneuver",
-    "network analysis",
-    "routing",
-    "sequence",
-    "stop order",
-    "stops"
-  ],
-  "relevant_apis": [
-    "DirectionManeuver",
-    "PolygonBarrier",
-    "Route",
-    "RouteParameters",
-    "RouteResult",
-    "RouteTask",
-    "Stop"
-  ],
-  "snippets": [
-    "src/main/java/com/esri/samples/routing_around_barriers/RoutingAroundBarriersSample.java",
-    "src/main/java/com/esri/samples/routing_around_barriers/RoutingAroundBarriersController.java",
-    "src/main/resources/routing_around_barriers/main.fxml"
-  ],
-  "title": "Routing Around Barriers"
-}
\ No newline at end of file
+    "category": "Network analysis",
+    "description": "Find a route that reaches all stops without crossing any barriers.",
+    "ignore": false,
+    "images": [
+        "RoutingAroundBarriers.png"
+    ],
+    "keywords": [
+        "barriers",
+        "best sequence",
+        "directions",
+        "maneuver",
+        "network analysis",
+        "routing",
+        "sequence",
+        "stop order",
+        "stops",
+        "DirectionManeuver",
+        "PolygonBarrier",
+        "Route",
+        "RouteParameters",
+        "RouteResult",
+        "RouteTask",
+        "Stop"
+    ],
+    "redirect_from": [],
```

Comment:
another instance of the array needing an empty string to be on the safe side: there are a few more of these kicking around.